abr_control/controllers/signals/gravity.py

```diff
@@ -0,0 +1,26 @@
+class Gravity(Signal):
```

Comment:
Not sure why this class was made, do we need it?

Reply:
wow throwback, from when we were bootstrapping learning. yeah we don't need this anymore